storage/manyrocks/manyrocks.go

```diff
@@ -229,6 +229,30 @@ func (t *ManyRocks) OpenExtent(id s.ExtentUUID, keyPattern s.KeyPattern, notify
 	}, nil
 }
 
+// ListExtents returns a list of extents available on the store
+func (t *ManyRocks) ListExtents() (extents []s.ExtentUUID, err error) {
```

Comment:
Can we have the contract return a tuple like (created_timestamp, extent_uuid, size) for each extent ?